akka-persistence-query/src/main/scala/akka/persistence/query/Offset.scala

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (C) 2015-2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.persistence.query
+
+import java.util.UUID
+
+trait Offset {
+  def hashCode: Int
```

Comment:
Ah I didn't realize case classes implement hasCode automatically. So I removed it in the new commit.